rush.json

```
@@ -324,7 +324,7 @@
     /**
      * The list of shell commands to run after the Rush build command finishes
      */
-    "postRushBuild": []
+    "postRushBuild": ["node apps/notifications/src/displayNotifications.js"]
```

```suggestion
    "postRushBuild": ["node apps/notifications/lib/displayNotifications.js"]
```
Which unfortunately is still problematic because there is no requirement that said project have been either installed or built in the local clone. The user is free to do `rush install --to rush-lib` followed by `rush build --to rush-lib`, for example. We'll need to prebundle a script to ensure that dependencies are satisfied.